pulsar-broker/src/main/java/org/apache/pulsar/broker/BookKeeperClientFactoryImpl.java

```diff
@@ -71,9 +71,7 @@ public BookKeeper create(ServiceConfiguration conf, MetadataStoreExtended store,
         MetadataDrivers.registerClientDriver("metadata-store", PulsarMetadataClientDriver.class);
 
         ClientConfiguration bkConf = createBkClientConfiguration(store, conf);
-        if (properties != null) {
-            properties.forEach((key, value) -> bkConf.setProperty(key, value));
-        }
+
```

Comment:
This seems like a breaking change. We are not able to set `bkConf` through `properties` at all.

Reply:
As the Isolation logic, if we set the isolation policy in bkConf,  when invoke  `newEnsemble` or `replaceBookie`  in `solatedBookieEnsemblePlacementPolicy` as  follow we also can use the policy setting to choice bookies, even we do not set the properties in here; It like do not cause breaking;
Cancel the properties setting in here is do not want the dynamic local policy covey the properties in broker.conf

https://github.com/apache/pulsar/blob/4c6989c4da6c0b18c9b0196630e03daf437cea68/pulsar-broker-common/src/main/java/org/apache/pulsar/bookie/rackawareness/IsolatedBookieEnsemblePlacementPolicy.java#L146-L160

Reply:
I understand that removing `properties` here can solve current isolation issue. My concern is that if this `properties` is introduced only to support isolation?

Reply:
@Jason918 This change satisfied our expect, the properties only contain bookie isolation properties. So remove the properties do not effect other properties be setting.

https://github.com/apache/pulsar/blob/cb305823b77ae36b67db575793a2f40f874a977a/pulsar-broker/src/main/java/org/apache/pulsar/broker/ManagedLedgerClientFactory.java#L92-L98

Reply:
If so, you need to remove the `properties` from the method.